spec/datadog/tracing/contrib/ethon/easy_patch_spec.rb

```diff
@@ -248,4 +248,15 @@
       expect(span.get_tag('out.host')).to eq('example.com')
     end
   end
+
+  context 'when query string in url' do
+    it 'does not collect string' do
+      easy = EthonSupport.ethon_easy_new(url: 'http://example.com/sample/path?foo=bar')
+
+      easy.perform
+
+      expect(span.get_tag('http.url')).to eq('/sample/path')
+      expect(span.get_tag('out.host')).to eq('example.com')
```

Comment:
I don't think we need to test for hostname here, given this test is asserting on query strings.